src/main/java/core/basesyntax/service/FileReaderImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> readFile(String fileName) {
+        List<String> data = new ArrayList<>();
+        try {
+            data = Files.readAllLines(Path.of(fileName));
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+        return data.subList(1, data.size() - 1);
```

Comment:
The method `readFile` returns `data.subList(1, data.size() - 1)`, which excludes the first and last lines of the file. Verify if this behavior is intentional. If the first line is a header and the last line is not needed, document this behavior clearly. Otherwise, consider returning the full list with `return data;`.